app/assets/stylesheets/fontastic-icons.scss

```diff
@@ -152,6 +152,9 @@
 .icon-icn-subject:before {
   content: "\e032";
 }
+.icon-icn-dna-check:before {
```

Comment:
Just noting that as of right now this icon represents a robot, not DNA. Whatever icon we choose, I recommend we stick with a class name that accurately describes the icon (so if we stick with a robot - update the name to reflect that)

Reply:
I reverted the icon back to the DNA check as discussed